e2e/actions/oci.go

```diff
@@ -1074,3 +1076,56 @@ func (c actionTests) actionOciOverlay(t *testing.T) {
 		})
 	}
 }
+
+// actionOciOverlayTeardown checks that OCI-mode overlays are correctly
+// unmounted even in root mode (i.e., when user namespaces are not involved).
+func (c actionTests) actionOciOverlayTeardown(t *testing.T) {
+	e2e.EnsureOCIArchive(t, c.env)
+	imageRef := "oci-archive:" + c.env.OCIArchivePath
+
+	mountInfoPath := "/proc/self/mountinfo"
```

Comment:
Nit: If there's a string that's definitely a constant, declare it `const` when possible, so that you don't ever accidentally change it.